app/views/users/_user_list.html.erb

```diff
@@ -1,40 +1,66 @@
-<div class="col-xs-12 col-sm-6">
-  <h2><%= t('projects.project_members') %></h2>
-  <ul class="list-group members-list">
+<div class="col-xs-12 col-sm-4">
+  <div class="row">
+    <div class="col-md-12">
+      <h4><i class="mi md-20">group</i> <%= t('projects.project_members') %></h4>
+    </div>
+
     <% @project.users.order('name').each do |user| %>
-      <%= content_tag :li, id: dom_id(user), class: 'list-group-item clearfix' do %>
-        <div class="member-name">
-          <%= user %>
-        </div>
+      <div class="col-md-12">
+        <div class="panel panel-default card user-card">
+          <div class="panel-body">
+            <div class="row">
+              <div class="col-xs-2">
+                <img src="data:image/png;base64,<%= RubyIdenticon.create_base64(user.username) %>" class="user-image">
```

Comment:
Is there some need to be an ident icon? why not just display user initials? Like:
```html
<span class="user-initials">MA</span>
```